lib/galaxy/tools/deps/docker_util.py

```diff
@@ -155,7 +155,7 @@ def build_docker_run_command(
     if terminal:
         command_parts.append("-t")
     for env_directive in env_directives:
-        command_parts.extend(["-e", shlex_quote(env_directive)])
+        command_parts.extend(["-e", env_directive])
```

Comment:
Is removing quoting intentional here?

Reply:
Yes - this was broken without that change. I'll rebase with a comment.